internal/xds/translator/testdata/out/xds-ir/http3.listeners.yaml

```diff
@@ -42,6 +42,7 @@
             alpnProtocols:
             - h2
             - http/1.1
+            - h3
```

Comment:
should we sort this?

Reply:
I thought of it, would change every file ..... I'm fine if you are

Reply:
we can do that in separated, let this one clean.